pkg/oc/oc.go

```diff
@@ -117,3 +120,65 @@ func CreateSecretsForTektonResults(){
 	//creating secret with generated certificate
 	cmd.MustSucceed("oc", "create", "secret", "tls", "-n", "openshift-pipelines", "tekton-results-tls", "--cert=cert.pem", "--key=key.pem")
 }
+
+func EnsureResutsReady(){
+	cmd.MustSucceed("oc", "wait", "--for=condition=Ready", "tektoninstallerset", "-l", "operator.tekton.dev/type=result")
+}
+
+func CreateResultsRoute(){
+	cmd.Run("oc", "create", "route", "-n", "openshift-pipelines", "passthrough", "tekton-results-api-service", "--service=tekton-results-api-service", "--port=8080")
+}
+
+func ConfigureResultsApi() string{
+	var results_api string = cmd.MustSucceed("oc", "get", "route", "tekton-results-api-service", "-n", "openshift-pipelines", "--no-headers", "-o", "custom-columns=:spec.host").Stdout()+":443"
+	results_api = strings.ReplaceAll(results_api, "\n", "")
+	return results_api
+}
+
+func GetAnnotaions(resorceType string) (string, string) {
```

Comment:
Function name indicates that it's a generic function which is not true - it returns Results annotations